cellassign_sum_factors.R

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env Rscript 
```

Comment:
The function you're wrapping here (computeSumFactors()) does not belong in the cellassign-cli package. We would instead put it in a separate scran-cli package. That would actually be a really useful thing to do, because scran is a useful tool to make available in other workflows.

Reply:
The issue with having it in this package is that when we want to use it in other workflows, we would have to load the cellassign-cli package, which would require cellassign and all its dependencies. If we factor it out into a separate scran-cli package, we only have the relatively small set of dependencies from that package to deal with.